src/Generators/ModuleGenerator.php

```diff
@@ -97,6 +114,20 @@ public function setPlain($plain)
         return $this;
     }
 
+    /**
+     * Set active flag.
+     *
+     * @param bool $active
+     *
+     * @return $this
+     */
+    public function setActive($active)
```

Comment:
Let's typehint the argument to boolean?

Reply:
I'd agree. But this argument comes from the Module class and it's not type hinted there, should I type hint it there too ?

Reply:
Hm, this can still be type hinted without changing the Module class no? I think that would be a first step.
This package existed before scalar type hints in php so I'm only now moving towards them over time.

Reply:
Ok, to be consistent with the Module class, I'll have to type hint them as integers even though they can only be 0 or 1. And they will have to be changed later, is that what you mean ?

Reply:
Indeed, now that I think of it, maybe there's no need to have them as int, as there's no other status other than 0 or 1. Meaning the could be booleans.
If possible you can typehint on both sides, if not, don't worry I'll do that later on myself.